app/models/label/LabelTable.scala

```diff
@@ -1012,6 +1014,19 @@ object LabelTable {
     _labels.list.map(l => LabelLocationWithSeverity(l._1, l._2, l._3, l._4.get, l._5.get, l._6, l._7, l._8, l._9, l._10))
   }
 
+  /**
+   * Returns all the submitted labels as well as the tags that were used with each label.
+   */
+  def selectTagsOfLabels(regionIds: List[Int]): List[LabelWithTags] = db.withSession { implicit session =>
+    val _labels = for {
```

Comment:
I think that this query should probably do all of the computation work, returning just one row for each tag. This is for performance reasons. For example, on the Seattle prod server the current implementation would mean reading the 200k labels into Scala objects (which means writing them to memory) and then transferring those 200k labels over the network to be processed on the front-end in JS. 

And since we're likely going to want to know this data in places other than the Admin page, I think it's important to do all the computation in SQL if we can! And we definitely can. The question is whether we can do it all in Slick!

As a reminder, [here's](https://github.com/tminglei/slick-pg/blob/master/core/src/main/scala/com/github/tminglei/slickpg/array/README.md) the link to the special operations for arrays, which I imagine is the trickiest part of this. I'm guessing that you'll want to use `unnest` to split each tag into it's own row. Then you should be able to group by label type and tag and do a count!

Reply:
Sounds good, I'll make those changes!

Reply:
<img width="1511" alt="Screenshot 2024-09-10 at 5 22 54 PM" src="https://github.com/user-attachments/assets/cfcd9fe9-b341-447a-8938-0da7961f9323">
This is the new result from the query which follows the format of {label_type: <label name>, tag: <tag name>, count: <num of tags>}.

Reply:
Nice, that looks great! And the query looks good to me at first glance!